chunky/src/java/se/llbit/math/structures/Position3d2IntPackedArray.java

```diff
@@ -57,7 +64,7 @@ public int get(int x, int y, int z) {
     if(arr != null) {
       return arr[packedIndex(x, y, z)];
     }
-    return 0;
+    return -1;
```

Comment:
Does this change affect any other existing code?

Reply:
I'm pretty sure the biome loading is the only user of this class so it shouldn't affect anything else